### PR TITLE
add site info for skipped file report

### LINKED ISSUE #7925: Additional info for skipped files
We want to get more info about the skipped files for a specific workflow.
As Alan pointed me, we can get the number of skipped files by task in here:
https://cmsweb.cern.ch/wmstatsserver/data/request/pdmvserv_HIG-PhaseISpring17DR-00009_00014_v0__170519_110139_2326

Additionally, we want to know where those files were skipped. So, by task, we want to get the number of skipped files per site.

### PR DESCRIPTION
**Pull request by ticoann**
fixes #7925

### CODE REVIEW

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
[Fix #7952 - Add site info for skipped file report](https://trello.com/c/pw3C1SDw/42-fix-7952-add-site-info-for-skipped-file-report)

**Comment by ticoann** [comment]
test this please

File: src/couchapps/FWJRDump/views/skippedFileInfoByTaskAndSite/map.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ function(doc) {
     }
     var specName = doc['fwjr'].task.split('/')[1];
     var task = doc['fwjr'].task;
-    
-    emit([specName, task], {"skippedFiles": doc['fwjr'].skippedFiles.length});
+    var site = "None";
```
Comment (amaltaro) [inline src/couchapps/FWJRDump/views/skippedFileInfoByTaskAndSite/map.js]: How about renaming it to something like "SiteNotReported" or so?

**Review by amaltaro — APPROVED** [review]
Looks good! This doesn't require any changes on wmstats, right?

**Comment by amaltaro** [comment]
Seangchan, can you fix that bold warning and also run a code reformat on the FWJR file?

**Comment by ticoann** [comment]
test this please

**Pull request merged**